advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,591 @@
+
+$ rm -rf /tmp/restored_data/*
+$ pgbackrest restore --stanza=demo --target="2021-04-19 14:38:55.313716+00:00" --type=time --no-delta --pg1-path=/tmp/restored_data
+HINT: time format must be YYYY-MM-DD HH:MM:SS with optional msec and optional timezone (+/- HH or HHMM or HH:MM) 
+	- if timezone is omitted, local time is assumed (for UTC use +00)
+P00   INFO: repo1: restore backup set 20210419-143643F
+
+$ rm -rf /tmp/restored_data/* 
+$ pgbackrest restore --stanza=demo --target="RP1" --type=name --no-delta --pg1-path=/tmp/restored_data --repo=2
+P00   INFO: repo2: restore backup set 20210419-143749F
+```
+
+Even if the backup in **repo2** is newer, the first match found for our time target is kept. Let's take a new backup in **repo1** to check if pgBackRest will auto-select the backup in **repo2**:
+
+```bash
+$ pgbackrest backup --stanza=demo --type=full --repo=1
+P00   INFO: backup command begin ...
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 00000001000000000000001B, lsn = 0/1B000028
+P00   INFO: full backup size = 63MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 00000001000000000000001B, lsn = 0/1B000138
+P00   INFO: check archive for segment(s) 00000001000000000000001B:00000001000000000000001B
+P00   INFO: new backup label = 20210419-144656F
+P00   INFO: backup command end: completed successfully
+P00   INFO: expire command begin ...
+P00   INFO: repo1: expire full backup 20210419-143643F
+P00   INFO: repo1: remove expired backup 20210419-143643F
+P00   INFO: expire command end: completed successfully
+
+$ pgbackrest info --stanza=demo
+stanza: demo
+    status: ok
+    cipher: none
+
+    db (current)
+        wal archive min/max (13): 000000010000000000000018/00000001000000000000001B
+
+        full backup: 20210419-143749F
+            timestamp start/stop: 2021-04-19 14:37:49 / 2021-04-19 14:38:03
+            wal start/stop: 000000010000000000000018 / 000000010000000000000018
+            database size: 63MB, database backup size: 63MB
+            repo2: backup set size: 10MB, backup size: 10MB
+
+        full backup: 20210419-144656F
+            timestamp start/stop: 2021-04-19 14:46:56 / 2021-04-19 14:47:10
+            wal start/stop: 00000001000000000000001B / 00000001000000000000001B
+            database size: 63MB, database backup size: 63MB
+            repo1: backup set size: 10MB, backup size: 10MB
+
+$ rm -rf /tmp/restored_data/*
+$ pgbackrest restore --stanza=demo --target="2021-04-19 14:38:55.313716+00:00" --type=time --no-delta --pg1-path=/tmp/restored_data
+P00   INFO: repo2: restore backup set 20210419-143749F
+```
+
+The recovery process may be complex and tricky depending on the target. So, the `info` command will really be helpful for you to chose the repository from which the data should be restored.
```

Comment:
The recovery process may be complex and tricky depending on the target, and therefore, the `info` command can really be helpful to determine the repository to restore from.